optlib/puppetManifest.ctags

```diff
@@ -39,6 +39,11 @@
 --kinddef-PuppetManifest=r,resource,resources
 --kinddef-PuppetManifest=v,variable,variables
 
+#
+# Separators
+#
+--_scopesep-PuppetManifest=/c:::
```

Comment:
Also why not add the `--_scopesep-PuppetManifest=*/*:::` to this file ?

Reply:
Oh, I see I should add it.